include/net/nrf_cloud_cell_pos.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef NRF_CLOUD_CELL_POS_H_
+#define NRF_CLOUD_CELL_POS_H_
+
+/** @file nrf_cloud_cell_pos.h
+ * @brief Module to provide nRF Cloud cellular positioning support to nRF9160 SiP.
+ */
+
+#include <zephyr.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @defgroup nrf_cloud_cell_pos nRF Cloud cellular positioning
+ * @{
+ */
+
+/** @brief Cell-based location request type */
+enum cell_based_location_type {
+	CELL_LOC_TYPE_INVALID = -1,
+	CELL_LOC_TYPE_SINGLE,
+	CELL_LOC_TYPE_MULTI /* Not yet supported */
+};
+
+/**@brief Request a cell-based location query from nRF Cloud.
+ *
+ * @param type        Type of cell-based location to request.
+ * @param request_loc If true, cloud will send location to the device.
+ *                    If false, cloud will not send location to the device.
```

Comment:
I'm probably missing some cell location knowledge here, but I don't understand this parameter, what's the point of requesting the locationrom the cloud if it's not supposed to be sent to the device?

Reply:
This is the original design by the nRF Cloud team, probably only to update device location on the cloud side.

Reply:
In many cases the device does not care about the location. but the cloud does

Reply:
The function is called `nrf_cloud_request_cell_location()`, but it only requests location is `request_loc` parameter is `true`. The name of the function could be changed, but I can't come up with a good name which would describe the functionality right now...

Also should we change the naming of functions so the that verb is the last word? For example `nrf_cloud_last_cell_location_get()`.

Reply:
to be consistent, with file naming convention, update to `nrf_cloud_cell_pos_request` or `rf_cloud_cell_pos_get` etc.

Reply:
renamed to nrf_cloud_cell_pos_request